From d4f20a4097f2dda79d35df9a87935d8d74021f50 Mon Sep 17 00:00:00 2001
From: Till Kampetter <till.kampetter@gmail.com>
Date: Tue, 9 Aug 2016 18:11:36 +0200
Subject: CUPS removes the "(recommended)" comments of the NickNames of
 Foomatic PPDs when listing available PPDs. This patch removes this removal
 action.

Last-Update: 2015-02-10

Patch-Name: reactivate_recommended_driver.patch
---
 scheduler/cups-driverd.cxx | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/scheduler/cups-driverd.cxx b/scheduler/cups-driverd.cxx
index 81c447ad8..d48c75165 100644
--- a/scheduler/cups-driverd.cxx
+++ b/scheduler/cups-driverd.cxx
@@ -243,7 +243,6 @@ add_ppd(const char *filename,		/* I - PPD filename */
 	const char *scheme)		/* I - PPD scheme */
 {
   ppd_info_t	*ppd;			/* PPD */
-  char		*recommended;		/* Foomatic driver string */
 
 
  /*
@@ -281,15 +280,6 @@ add_ppd(const char *filename,		/* I - PPD filename */
   strlcpy(ppd->record.device_id, device_id, sizeof(ppd->record.device_id));
   strlcpy(ppd->record.scheme, scheme, sizeof(ppd->record.scheme));
 
- /*
-  * Strip confusing (and often wrong) "recommended" suffix added by
-  * Foomatic drivers...
-  */
-
-  if ((recommended = strstr(ppd->record.make_and_model,
-                            " (recommended)")) != NULL)
-    *recommended = '\0';
-
  /*
   * Add the PPD to the PPD arrays...
   */
